packages/sources/vue/src/components/indicators/VtmnProgressbar/VtmnProgressbar.vue

```diff
@@ -69,9 +81,9 @@ export default /*#__PURE__*/ defineComponent({
   <div
     :class="classes"
     role="progressbar"
-    :aria-valuemin="0"
-    :aria-valuemax="100"
-    :aria-valuenow="value"
+    :aria-valuemin="status === 'determinate' ? 0 : undefined"
+    :aria-valuemax="status === 'determinate' ? 100 : undefined"
+    :aria-valuenow="status === 'determinate' ? value : undefined"
     :aria-labelledby="labelId ? labelId : undefined"
```

Comment:
For spreading `ariaProps` you can investigate: https://github.com/vuejs/core/issues/4105